packages/xo-web/src/xo-app/backup/new/new-schedule.js

```diff
@@ -15,6 +15,7 @@ import { FormGroup, Input } from './../utils'
 const New = decorate([
   provideState({
     computed: {
+      forceFullInterval: (_, { value }) => value.fullInterval === 1,
```

Comment:
Why send `_` as parameter if he is not used ?

Reply:
Reaclette pass the component state as the first argument and the component props as second argument.

This definition is similar than:
```js
{
  forceFullInterval: (...args) => args[1].fullInterval === 1
}
```